.github/workflows/build.yml

```diff
@@ -39,6 +39,12 @@ jobs:
                 os: ubuntu-latest
               - projectloadstyle: net46
                 os: ubuntu-latest
+              - projectloadstyle: net35
```

Comment:
don't you need to add the os to the matrix?

Reply:
I had it, just forgot to copy over to the new file.